slack/slack.go

```diff
@@ -0,0 +1,73 @@
+package slack
```

Comment:
Why we have this in common-libraries?

Reply:
I thought of adding this here, considering other services or modules could re-use this.
They will just need to set their own custom webhook url.

Reply:
Will be same for all